src/Exception/ExceptionInterface.php

```diff
@@ -0,0 +1,11 @@
+<?php
+
+namespace Bernard\Exception;
+
+/**
+ * Interface for exceptions thrown by Bernard
+ * @package Bernard
+ */
+interface ExceptionInterface
```

Comment:
We dont't use the `Interface` suffix in Bernard for interfaces. http://verraes.net/2013/09/sensible-interfaces/